packages/valory/connections/ledger/tests/test_ledger_api.py

```diff
@@ -638,7 +638,7 @@ async def test_no_raw_tx(
     @pytest.mark.asyncio
     @pytest.mark.parametrize(
         "failing_ledger_method_name",
-        ("get_transaction_receipt", "is_transaction_settled", "get_transaction"),
+        ("get_transaction_receipt", "get_transaction"),
```

Comment:
Some checks below do not make sense anymore, e.g.:
https://github.com/valory-xyz/open-aea/blob/154eb91729a4fbe29feaf3006e8dff641f224738/packages/valory/connections/ledger/tests/test_ledger_api.py#L676